src/components/SearchRating.vue

```diff
@@ -49,20 +49,19 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-@import "@creativecommons/vocabulary/scss/color.scss";
-
 .rating {
   vertical-align: middle;
-  color: $color-dark-turquoise;
-  font-size: 0.8rem;
-  text-decoration: none;
+  color: rgb(5, 181, 218) !important;;
```

Comment:
you have a double semi-colon here.
also.. why are so many `!important` here?

Reply:
because of the issue in #829 

This is just a quick workaround because fixing the issue will take some time I think